### ISSUE TITLE
Make Stock Ticker Thing Look Super Fancy

### ISSUE THREAD

**Issue by mathfour** [Tfeinstein24/Oilfield-Calculations#12]
Choose a fancy api to add to the site and implement it.

**Comment by mathfour** [comment]
I'm not sure what some of the code in the js file is, but it seems it all works in the console. Now it just needs to find it's way to the page itself. 

Good job, TF!

**Comment by Tfeinstein24** [comment]
Thanks, it's on the page in my branch. Good job ok the calcs by the way
they look great.

**Comment by mathfour** [comment]
I see you have `<script type="text/javascript" src="StockTickerAPI.js"></script>` in the html, but that doesn't put it on the page. Am I missing something?

**Comment by Tfeinstein24** [comment]
Yeah it should link to the js file...I will be done early today so I can
take a look at it. It's working on my computer

**Comment by mathfour** [comment]
![image](https://cloud.githubusercontent.com/assets/7103316/25501447/6241c4a8-2b58-11e7-880f-3fdb68bfb839.png)
